permissions/MediaIndexing.permission

```diff
@@ -20,3 +20,6 @@ dbus-user.broadcast org.freedesktop.Tracker1=org.freedesktop.Tracker1.*@/*
 dbus-user.talk org.freedesktop.Tracker1.*
 dbus-user.broadcast org.freedesktop.Tracker1.*=org.freedesktop.Tracker1.*@/*
 # END sessionbus-org.freedesktop.Tracker1.resource
+
+dbus-user.talk org.freedesktop.Tracker3.Miner.Files
```

Comment:
We should probably now keep checking if there is some reason to limit access via dbus-user.call. This gives access to everything which can be okay for some services but it's sometimes too lax.

Reply:
Yea, and actually the whole new tracker is now more aimed for sandboxing. But so far I've aimed more to getting the new version running properly and haven't bothered with the permission side too much. Thinking this shouldn't probably be much of new security problem so could be an intermediate step.